tests/queries_tests/tag_test.js

```diff
@@ -0,0 +1,41 @@
+import chai, { expect } from 'chai'
+import * as tag from '../../src/database/queries/tag'
+
+describe('tag', () => {
```

Comment:
the mocha docs highly recommend you avoid passing `describe`, `context, and `it` with an `=>` function. 

use:
```js
describe('tag', function(){
```